app/controllers/sessions_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
     redirect_params = {}
     if user.nil?
       redirect_params = { alert: 'Login or password incorrect. Try again.' }
+    elsif user.email_confirmed_at.nil?
```

Comment:
So once in production we force automatically email validation?
Should we have a site setting for how to treat non new accounts?